src/mut/section.cpp

```diff
@@ -203,57 +202,69 @@ std::shared_ptr<Section> Section::appendSection(
     return ptr;
 }
 
-bool Section::_compare(const Section& other, bool verbose) const
+bool Section::diff(const Section& other, bool verbose) const
 {
     if (this->type() != other.type()) {
         if (verbose)
-            std::cout << "Section type differ" << std::endl;
-        return false;
+            std::cout << "Reason: section type differ" << std::endl;
```

Comment:
maybe `std::cerr`?

Reply:
I am not sure. The message is what is expected to be print out, it is not really an error.

Reply:
`stderr` is not necessarily for errors, but for additional "diagnostics" as well.
But now that I took a look at some `diff`-ing tools, all of them use `stdout`, so nvm.